src/code/audio/audio_dcache.c

```diff
@@ -0,0 +1,15 @@
+#include "global.h"
+
+void Audio_InvalDCache(void* buf, s32 size) {
```

Comment:
Hmm so this is a cache size. Thinking size_t is the appropriate type here. Anyone else have any input?

```suggestion
void Audio_InvalDCache(void* buf, size_t size) {
```